### ISSUE TITLE
Revise logical definition for feline leukemia

### ISSUE THREAD

**Issue by diatomsRcool** [monarch-initiative/mondo#5567]
**Mondo term (ID and Label)**
leukemia, feline MONDO:0025488

**Suggested revision of logical definition**
non-human animal disease and infectious disease and (disease has primary infectious agent some Viruses) and (in taxon some Felis)

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)
0000-0002-2908-3327

**Comment by sabrinatoro** [comment]
refer to https://github.com/monarch-initiative/mondo/issues/5566

Also, it sounds like (assuming we are continuing to use the "cross-species analog" relation), we could have 2 logical definitions. @nicolevasilevsky, is it something that is allowed?

**Comment by sabrinatoro** [comment]
based on Katie's comment on #5566, this suggested logical definition would be incorrect, as not all feline leukemia are due to a virus infection.

**Comment by nicolevasilevsky** [comment]
> Also, it sounds like (assuming we are continuing to use the "cross-species analog" relation), we could have 2 logical definitions. @nicolevasilevsky, is it something that is allowed?

no, unfortunately

**Comment by matentzn** [comment]
In general we should 

- define the human "analog"
- link the species-specific class to the human analog with the cross-species analog relation

However this pattern will eventually break down, in particular where we have non-human diseases without a human counterpart.

Mondo Technical Call material.

**Comment by diatomsRcool** [comment]
Yes, I think maybe we need to set this one aside as "something that needs to be discussed in a workshop"

**Comment by nicolevasilevsky** [comment]
@diatomsRcool can I convert this to a discussion topic since this does not seem to be an actionable ticket?

**Comment by diatomsRcool** [comment]
Fine by me.
